spec/features/comments_feature_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+feature 'Comments' do
+
+  before(:each) do
+    @user = create(:user)
+    @user2 = create(:user, username:'some_user', email: 'test2@email.com')
+  end
+
+  context 'commenting on a picture' do
+
+    scenario 'cannot add a comment when not logged in' do
+      @user.pictures.create(caption: 'coding at home', name: 'Nightlife', image_file_name: 'nightlife.png')
```

Comment:
Metrics/LineLength: Line is too long. [107/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)